brain/python/default_docstrings.cpp

```diff
@@ -0,0 +1,97 @@
+// File automatically generated by /home/hernando/dev/src/config.stable/Pydoxine/CMake/../docstrings/sphinx_ext/cpp_docstrings.py
```

Comment:
I had this discussion with Stefan. The problem is that without Sphinx >= 1.3 the init_docstrings.cpp file won't be generated and in CI we have 1.2.2. I added this one as a fallback, the problem is that it needs to be updated manually.
I can skip adding docstrings to the wrapping if Sphinx is not found, but if you look closely to the rest of this PR you will realize that is not that simple to just let Boost.Python do its default thing if the docstrings are not there (instead of replacing everything with empty strings).